pytorch_lightning/accelerators/accelerator_connector.py

```diff
@@ -343,7 +343,7 @@ def set_distributed_mode(self):
         if self.trainer.num_nodes > 1 and not (self.trainer.use_ddp2 or self.trainer.use_ddp):
             raise MisconfigurationException(
                 'DataParallel does not support num_nodes > 1. Switching to DistributedDataParallel for you. '
```

Comment:
```suggestion
                'DataParallel does not support num_nodes > 1. '
```
since it's an exception, it won't be switching it right?